File: app/src/main/java/com/ianfield/bodyscoring/utils/Setting.java
package com.ianfield.bodyscoring.utils;

import android.support.annotation.StringDef;

/**
 * Created by Ian Field on 05/08/2016.
 */
public class Setting {
    public static final String UK = "UK";
    public static final String NZ = "NZ";

    @StringDef({
            UK,
            NZ,
    })
    public @interface Country {
    }
}

File: app/src/main/java/com/ianfield/bodyscoring/utils/Setting.kt
package com.ianfield.bodyscoring.utils

import androidx.annotation.StringDef

/**
 * Created by Ian Field on 05/08/2016.
 */
class Setting {

    companion object {
        const val UK = "UK"
        const val NZ = "NZ"
    }


    @StringDef(UK, NZ)
    @Retention(AnnotationRetention.SOURCE)
    annotation class Country
}
